wordpress-mobile/WordPress-Android
# [Gutenberg] Fix Media & Text block crash

`Gutenberg` PR -> https://github.com/WordPress/gutenberg/pull/18907

Fixes https://github.com/wordpress-mobile/WordPress-Android/issues/10876 Media & Text crash.

**NOTE**: This PR is _just for testing_ the fix, **do not merge**.

To test:
- Download the generated APK in this PR.
- Open the Gutenberg **Web** editor:
- Create a new post with this code:
```

<div class="wp-block-media-text alignwide is-stacked-on-mobile is-image-fill"><figure class="wp-block-media-text__media" style="background-image:url(https://geriuxx-atomic-site-1.blog/wp-content/uploads/2019/12/img_0004-1.jpg);background-position:52% 12%"><img src="https://geriuxx-atomic-site-1.blog/wp-content/uploads/2019/12/img_0004-1.jpg" alt="" class="wp-image-29"/></figure><div class="wp-block-media-text__content">
<p class="has-large-font-size"></p>
</div></div>

```
- Save
- Open the newly created post from the **app without metro running**
- Expect the `Media & Text` block to be visible without any crashes.

PR submission checklist:
